test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -2029,7 +2029,7 @@ private void TestPipeline(IEstimator<ITransformer> pipeline, IDataView dataView,
         private void TestPipeline<TLastTransformer>(EstimatorChain<TLastTransformer> pipeline, IDataView dataView, string onnxFileName, ColumnComparison[] columnsToCompare, string onnxTxtName = null, string onnxTxtSubDir = null)
             where TLastTransformer : class, ITransformer
         {
-            var model = pipeline.Fit(dataView);
+            using var model = pipeline.Fit(dataView);
```

Comment:
might want to update the samples inside the docs folder as well.

Reply:
Done.

---
In reply to: [533779757](https://github.com/dotnet/machinelearning/pull/5518#discussion_r533779757) [](ancestors = 533779757)